components/docs-chef-io/content/automate/flags.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Feature Flags"
+
+draft = false
+[menu]
+  [menu.automate]
+    title = "Feature Flags"
+    parent = "automate/reference"
+    identifier = "automate/reference/flags.md Feature Flags"
+    weight = 50
++++
+
+[\[edit on GitHub\]](https://github.com/chef/automate/blob/master/components/docs-chef-io/content/automate/flags.md)
+
+Use the beta, lega, or feat flags to turn on or off beta, legacy, or in-development features.
+
+Select anywhere on the Chef Automate interface and enter "feat", "lega", or "beta" to open the feature flags window.
```

Comment:
```suggestion
Select anywhere on the Chef Automate interface and type "feat", "lega", or "beta" to open the feature flags window.
```

i think we may need to be specific that they need to type in those letter sequences

Reply:
@susanev Can you put in "feat", "lega", or "beta" via text-to-speech in Chef Automate UI?

Reply:
prolly yah. so types not the right word. but i think we need to say something like input the letter sequences though. ive tried to explain this thing to so many users and most are like where? i dont see beta anywhere. and i had to be like no, you literally type those characters. idk, am i making any sense?

Reply:
@jwdean that change was to prevent the modal from coming up when people are searching for nodes with the characters beta, feat, and lega in the name. can you say more about what clarification you would like added?

Reply:
@susanev Sorry, new to this and wasn't clear. My thought was only that "Select anywhere" isn't exactly correct either. In my mind I think "click in any whitespace", more irritatingly "click anywhere that isn't the search box", "with the cursor not in a search box." Nothing seems terribly elegant.

Reply:
Maybe "click outside the search box and type the letters F-E-A-T, etc"?

Reply:
Good work @mjingle! These are solid choices for describing the process in accessible terms, and these choices are in keeping  with the current practices in technical documentation. "Select" is the preferred term for engaging with a UI and "enter" is the preferred term for how a user adds information to the UI. Mary already considered the alternatives suggested above. "Type" is inaccurate (& exclusionary) in the case of voice-to-text and "input" is reserved for use as part of the procedural action pair "input/output".

Reply:
@kagarmoe Thank you for the standards clarification. That is very informative. No offence intended, simply trying to suggest something that was clear selecting in the search field is no longer a valid option.

Reply:
Thanks @jwdean! No worries--we generally enjoy a good docs debate! I think the discussion illustrates that we need to sharpen the interaction.